src/NuGet.Services.AzureSearch/Models/SearchDocument.cs

```diff
@@ -59,6 +63,25 @@ public class UpdateVersionList : KeyedDocument, IVersions
         public interface IVersions : IKeyedDocument
         {
             string[] Versions { get; set; }
+            bool? IsLatestStable { get; set; }
+            bool? IsLatest { get; set; }
+        }
+
+        /// <summary>
+        /// The data required to populate <see cref="IVersions"/> and other <see cref="SearchDocument"/> classes.
+        /// </summary>
+        public class LatestFlags
```

Comment:
From my understanding, these are the `LatestFlags` for a specific search filter. If so, please mention that in the comment to add a little more context

Reply:
All `SearchEntities` are per search filter. Sure, i'll add a comment.